packages/cli-repl/src/completer.ts

```diff
@@ -0,0 +1,76 @@
+import { types as shellTypes } from 'mongosh-shell-api';
+/**
+ * Return complete suggestions given currently typed line
+ *
+ * @param {any} output - The output.
+ *
+ * @returns {string} The output.
+ */
+function completer(line: string): any {
```

Comment:
Isn't return type 'string' according to jsdoc?

Reply:
i couldn't get the compiler to compile, and `any` worked. I changed the `typescript` definition to `[string[], string]` now as it's more specific.